views/project/invoice.erb

```diff
@@ -70,6 +70,12 @@
               <%= @invoice_data[:billing_state] %>
               <%= @invoice_data[:billing_postal_code] %>,<br>
               <%= @invoice_data[:billing_country] %><br>
+              <% if !@invoice_data[:tax_id].nil? %>
+                <%= @invoice_data[:tax_id] %> <br>
```

Comment:
IMHO,
- Move tax ID to the top of the address block
- Add a tax ID prefix to it
- Move company name before the billing name
```suggestion
                Tax ID: <%= @invoice_data[:tax_id] %> <br>
```

before:
<img width="1227" alt="Screenshot 2024-02-28 at 21 26 40" src="https://github.com/ubicloud/ubicloud/assets/993199/5a6cd43d-6c7f-4460-b453-8024a4aff9a3">

after:
<img width="1258" alt="Screenshot 2024-02-28 at 21 36 06" src="https://github.com/ubicloud/ubicloud/assets/993199/eda014a3-98f4-4cd3-824f-1f13c6c3faa9">